src/ContainerDetails.jsx

```diff
@@ -17,13 +17,18 @@ const render_container_published_ports = (ports) => {
     if (!ports)
         return null;
 
-    const result = [];
+    const result = ports.map(port => {
+        const protocol = port.protocol;
+        const host_port = port.hostPort || port.host_port;
+        const container_port = port.containerPort || port.container_port;
+        const host_ip = port.hostIP || port.host_ip || '0.0.0.0';
```

Comment:
Do we want to have a comment on when this was introduced and for podman 4.00

Reply:
Added comment